### PR TITLE
fix(Studio installation): update Customer Service Center links

### PR DESCRIPTION
**Pull request by ugaston**
Update links to Customer Service Center

Versions: 2022.2, 2023.2, 2024.1, 2024.2

### CODE REVIEW

**Review by rbioteau — APPROVED** [review]

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 = Release notes
-:description: Bonita release note
+:description: This is the release notes for Bonita {bonitaVersion} versions
```
Comment (tbouffard) [inline modules/ROOT/pages/release-notes.adoc]: **suggestion:** 
```suggestion
:description: This is the release notes for Bonita {bonitaVersion} version
```

**Comment by tbouffard** [comment]
Converted to draft, while I check the preview site

File: antora.yml [hunk 2]
```diff
@@ -8,6 +8,9 @@ asciidoc:
     bonitasoft-docker-repository: '{bonitasoft-registry}/docker'
     bonitaTechnicalVersion: '7.15.0' # latest public community version
     javadocVersion: 7.15
+    cscRootUrl: https://csc.bonitacloud.com/apps/CustomerServices
```
Comment (tbouffard) [inline antora.yml]: **nitpick:** wrong URL, we will fix this with @ugaston

**Review by tbouffard — CHANGES_REQUESTED** [review]

File: antora.yml (hunk 2 above)
Comment (ugaston) [reply antora.yml]: Good catch!

Indeed, the good URL should read: csc.bonitacloud.**bonitasoft**.com

Thanks for the review.

Comment (tbouffard) [reply antora.yml]: fixed in 62c07e0

**Review by tbouffard — APPROVED** [review]

**Pull request merged**